### DISCUSSION TITLE
Does OB support Mac M1 arm64 archtecture

### DISCUSSION THREAD

**Discussion by sandszhouSZ** [oceanbase/oceanbase#1097] [General]
Does OB support Mac M1 arm64 archtecture

**Comment by sandszhouSZ** [comment]
这里有个docker 镜像的方案，https://ask.oceanbase.com/t/topic/31400008

但是我期望一个能在mac裸机器上进行编译的环境

**Comment by caifeizhi** [comment]
ob支持arm64，但暂不支持Mac操作系统，暂时还是需要通过docker的方式进行；Mac操作系统的兼容预计明年启动
